CODEOWNERS

```diff
@@ -11,25 +11,26 @@ k8s/aat/ @hmcts/platform-engineering @hmcts/devops
 ## environment rules
 k8s/prod/ @hmcts/production-apps-approvals
 
-## application rules
+## Platform rules
 
 k8s/common/ @hmcts/platform-engineering @hmcts/devops
 k8s/namespaces/ @hmcts/platform-engineering @hmcts/devops
-k8s/**/cluster-*-overlay/ @hmcts/platform-engineering
+k8s/**/*-overlay/ @hmcts/platform-engineering
```

Comment:
This will make sure all files in overlays are control by platform. So, anyone adding a new overlay ( even for sandbox should need approval). Do we want to avoid sandbox ? Note: they can still add any manifests in their namespace, but we control what goes in what environment.